libkbfs/crypto_common.go

```diff
@@ -202,9 +222,18 @@ func (c CryptoCommon) EncryptBlock(block Block, key kbfscrypto.BlockCryptKey) (
 
 // DecryptBlock implements the Crypto interface for CryptoCommon.
 func (c CryptoCommon) DecryptBlock(
-	encryptedBlock kbfscrypto.EncryptedBlock, key kbfscrypto.BlockCryptKey,
-	block Block) error {
-	paddedBlock, err := kbfscrypto.DecryptBlock(encryptedBlock, key)
+	encryptedBlock kbfscrypto.EncryptedBlock,
+	tlfCryptKey kbfscrypto.TLFCryptKey,
+	blockServerHalf kbfscrypto.BlockCryptKeyServerHalf, block Block) error {
+	var paddedBlock []byte
+	var err error
+	if encryptedBlock.UsesV2() {
```

Comment:
this case does seem to be simpler if there were a `DecryptBlock(..., version)` function in kbfscrypto, since you'd save having to look at the version twice

Reply:
Will fix in a future cleanup PR.